aips/aip-51.md

```diff
@@ -0,0 +1,65 @@
+---
+
+aip: 51
+
+title: Changing beneficiary addresses
+
+author: junkil-park, michelle-aptos, movekevin, wintertoro
+
+discussions-to: https://github.com/aptos-foundation/AIPs/issues/251
+
+Status: In Review
+
+type: Standard Framework
+
+created: 10/3/2023
+
+---
+
+
+# AIP-51 - Changing beneficiary addresses
+
+## Summary
+
+Right now, the operator commission goes to the operator's address. We have received multiple requests for the ability to set a different beneficiary address. One example use case is when operators want their commission to go to a cold storage wallet while their operator address is used for day-to-day operations. This feature should apply to both the staking_contract and the delegated_staking contract.
+
+
+### Goals
+
+The goal is to allow operators to set a different beneficiary address for their commission rewards. So, the following scenarios should be possible:
+- operators want to store commission rewards in a cold wallet. This change would introduce the decoupling of the wallet and operator keys so that funds can be sent to an address different from the operator's account.
+- The funds would remain in a cold wallet while the operator can perform other operations in a hot wallet.
+
+
+### Out of Scope
+
+Changing the beneficiaries for token owners is out of the scope of this AIP.
+
+
+### Impact
+
+This feature impacts operators and their beneficiaries.
```

Comment:
Could you please callout that the default value for beneficiary will be set to operator address? so they should expect no behavior change if they did not take any action when we release this new feature

Reply:
Yes, I updated accordingly. The beneficiary of any operator would be the operator itself if the operator didn't set the beneficiary explicitly.